### PR TITLE
TYP: misc

### PR DESCRIPTION
**Pull request by twoertwein**

### CODE REVIEW

File: pandas/core/generic.py [hunk 1]
```diff
@@ -6608,7 +6608,7 @@ def replace(
         inplace: bool_t = False,
         limit: int | None = None,
         regex=False,
-        method=lib.no_default,
+        method: str | lib.NoDefault = lib.no_default,
```
Comment (mroeschke) [inline pandas/core/generic.py:6611]:
```suggestion
        method: Literal["pad", "ffill", "bfill"] | lib.NoDefault = lib.no_default,
```

I think those are the possible methods

Comment (twoertwein) [reply pandas/core/generic.py:6611]: Thank you! `None` seems also to be accepted based on the doc-string.

Comment (twoertwein) [reply pandas/core/generic.py]: I removed `None` from the documentation - it raises errors when using it.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @twoertwein

**Pull request merged**